tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -390,6 +436,51 @@ def _create_fsx_open_zfs_volume_ids(num_existing_fsx_open_zfs_volumes, fsx_facto
     return []
 
 
+def _get_fsx_file_cache_id(num_existing_fsx_file_caches, create_fsx_file_cache, file_cache_path, bucket_name):
```

Comment:
This function is expected to return a single FileCache id, so what about returning None rather than an empty list?

Reply:
I will make these changes as Initially the code was written so that i add Fsx FC in ```test_multiple_fsx``` rather than as a standalone.